### PR TITLE
2231 semester changes.

### PR DESCRIPTION
**Pull request by nll5171**
I fumbled the bag a few times, that's why there's 4 commits. Don't worry about it, I'm good at my job I swear.

### CODE REVIEW

**Review by nogarremi — CHANGES_REQUESTED** [review]
Please delete `.vs/RIT FGC Website Fork/v17/.wsuo` and  `.vs/VSWorkspaceState.json` and add `**/.vs` to .gitignore on a new line

File: index.html [hunk 1]
```diff
@@ -124,34 +124,34 @@ <h3>THE CLUB</h3>
...
 					<div class="col-sm-4">
 						<p class="text-center">
-							<strong>Guilty Gear -STRIVE-</strong>
+							<strong>Street Fighter 6</strong>
 						</p>
 						<br>
-						<a href="#ggstRanks" data-toggle="collapse">
-							<img id="ggst" src="resources/imgs/ggst.png" class="person" alt="Guilty Gear -STRIVE-" width="255" height="255"/>
+						<a href="#sf6Ranks" data-toggle="collapse">
+							<img id="rev2" src="resources/imgs/sf6.png" class="person" alt="Street Fighter 6" width="255" height="255"/>
```
Comment (nogarremi) [inline index.html:140]: The ID field here is set to `rev2`

Comment (nogarremi) [reply index.html:140]: Should be `sf6`

**Review by nogarremi — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (nll5171) [reply index.html]: Changes have been made as requested.

**Comment by nll5171** [comment]
> Please delete `.vs/RIT FGC Website Fork/v17/.wsuo` and `.vs/VSWorkspaceState.json` and add `**/.vs` to .gitignore on a new line

Went ahead and deleted said files, as well as updated the .gitignore. Apologies for the trouble, that slipped my mind to do.

**Review by nogarremi — APPROVED** [review]

**Pull request merged**